cumulusci/tasks/salesforce/CreateCommunity.py

```diff
@@ -119,5 +120,5 @@ def _poll_action(self):
 
     def _get_community(self):
         community_list = self.sf.restful("connect/communities")["communities"]
-        communities = {c["name"]: c for c in community_list}
-        return communities.get(self.options["name"])
+        communities = {c["name"].casefold(): c for c in community_list}
```

Comment:
We use .lower() everywhere else. Is there anything special about this context?

Reply:
Nothing special here. I just stumbled on it in the docs and thought "huh, better caseless matching of unicode strings" and added it here.

Reply:
Following up, @prescod. Are you requesting a change here?